frontend/views/containers/proposals/PropositionsAllModal.vue

```diff
@@ -55,10 +55,23 @@ export default ({
   async mounted () {
     const key = `proposals/${this.ourIdentityContractId}/${this.currentGroupId}`
     this.ephemeral.proposals = await sbp('gi.db/archive/load', key) || []
+
+    this.checkTargetAndScroll()
   },
   methods: {
     unfocusSelect () {
       this.$refs.select.blur()
+    },
+    checkTargetAndScroll () {
+      const targetId = this.$route.query?.targetProposal || ''
+
+      if (targetId && this.ephemeral.proposals.some(entry => entry[0] === targetId)) {
+        setTimeout(() => {
+          const targetEl = this.$refs.pList.querySelector(`[data-proposal-hash="${targetId}"]`)
+
+          targetEl && targetEl.scrollIntoView({ block: 'center' })
+        }, 150)
```

Comment:
Is there a way to do this without a timeout?

Reply:
The delay is actually not necessary here. So replaced it with `nextTick()` instead.